experimental/pedometer/jnigen.yaml

```diff
@@ -3,8 +3,6 @@ android_sdk_config:
   add_gradle_sources: true
   android_example: 'example/'
 
-suspend_fun_to_async: true
```

Comment:
I checked the changelog to understand this and it's cool to see you can now reliably determine this necessity.